src/index.html

```diff
@@ -3,12 +3,55 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect"
+    href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a href="#home" class="logo">
+          <img src="https://github.com/shumenyatko/layout_moyo-header/blob/master/src/images/logo.png?raw=true" alt="Moyo">
+        </a>
+        <div class="navigation">
+          <nav>
+            <ul>
+              <li>
+                <a href="#" class="is-active"
+                data-qa="hover">APPLE</a>
```

Comment:
```suggestion
                <a 
                  href="#" 
                  class="is-active"
                  data-qa="hover"
                >
                  APPLE
                </a>
```
correct formatting - read the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

Also, pay attention to where exactly you should add `data-qa` attributes (read task requirements)